content/en/certifications/_index.md

```diff
@@ -9,5 +9,5 @@ images: []
 weight: 100
 layout: default
 ---
-[LRA 1.0 Compatible Certification Request (CCR)](lra/microprofile-lra-1.0/index.html)
+[LRA 2.0 Compatible Certification Request (CCR)](https://github.com/jbosstm/narayana/blob/main/certifications/lra/microprofile-lra-2.0/README.adoc)
```

Comment:
The zips in the `lra/microprofile-lra-2.0/` directory refer to lra-1.0. Will you be able to look into that?

Reply:
Yes, I will update those zips.

Reply:
I opened a PR for it (https://github.com/jbosstm/narayana/pull/2161)